get_dependencies.py

```diff
@@ -24,18 +24,40 @@ def download(url, filename):
     sys.stdout.write('\n')
 
 
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if _platform == "linux" or _platform == "linux2":  # linux
+    osys = "linux"
+elif _platform == "darwin":    # MAC OS X
+    osys = "mac"
+
+
+if osys == "linux":
+    print('linux os detected !')
+    print('[*] Downloading support files for linux ~ 108 mb')
+    download('https://www.dropbox.com/s/3mosdy9tsf7pilh/linux.zip?dl=1', 'linux.zip')
```

Comment:
Let's try to avoid code duplication, the Linux and Mac code is almost the same, it doesn't make sense to rewrite it twice.

Reply:
directory for the bin files in both the cases are different.

Reply:
You can use a variable to store the path of the directory.

Reply:
i used it collect.py ,  and also since 'Google Chrome' contains a space in between therefore it was not working even i  tried /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome but it didn't work , so i  had to change the bin file name to 'chrome' then it started working

Reply:
I think this is not related to what I asked? I only asked to avoid code duplication between the `linux` and `mac` branches.
E.g. instead of:
```
if linux:
  download(URL_LINUX, NAME_LINUX)
elif mac:
  download(URL_MAC, NAME_MAC)
```
You can do:
```
if linux:
    url = URL_LINUX
    name = NAME_LINUX
elif mac:
    url = URL_MAC
    name = NAME_MAC

download(url, name)
```

Reply:
okay

Reply:
then sir, we will have to use one more if else condition for extracting the files.

Reply:
Not if we manage to use ZipFile on Mac too (https://github.com/marco-c/autowebcompat/pull/34#discussion_r165985224).